src/analysis/retail/shaman/elemental/modules/talents/Stormkeeper.tsx

```diff
@@ -58,19 +518,48 @@ class Stormkeeper extends Analyzer {
     );
   }
 
-  guideSubsection(): JSX.Element {
-    const explanation = <>TODO</>;
-    const data = this.statistic();
-
+  description() {
     return (
-      <ExplanationAndDataSubSection
-        title="Stormkeeper"
-        explanationPercent={GUIDE_EXPLANATION_PERCENT_WIDTH}
-        explanation={explanation}
-        data={data}
-      />
+      <>
+        <p>
+          <strong>
+            <SpellLink spell={TALENTS.STORMKEEPER_1_ELEMENTAL_TALENT} />
+          </strong>{' '}
+          massively amplifies the strength of your next two
+          <SpellLink spell={SPELLS.LIGHTNING_BOLT} /> or
+          <SpellLink spell={TALENTS.CHAIN_LIGHTNING_TALENT} />
+          s. Therefore, you want to combine this effect with as many of the other damage amplifying
```

Comment:
[Do] For consistency of pluralization you want either "Lightning Bolts or Chain Lightnings" or "Lightning Bolt or Chain Lightning Casts".